Templates/HttpTrigger-Powershell/function.json

```diff
@@ -0,0 +1,16 @@
+{
+    "disabled": false,
+    "bindings": [
+        {
+            "authLevel": "function",
```

Comment:
This is the default auth level - doesn't need to be specified. Our other http samples don't specify it.